src/graphql/models/Article.js

```diff
@@ -240,6 +240,8 @@ const Article = new GraphQLObjectType({
         });
       },
     },
+
+    _highlight: { type: GraphQLString },
```

Comment:
I think `highlight` should be added to `ArticleEdge` and `ReplyEdge` instead.

- `highlight` is only meaningful for listing APIs such as `ListArticles` and `ListReplies`. Like `score` and `cursor`, it is an attribute that is related to the relation of the filter operation, thus the properly should belong to edge object.
- In answerfamily I added `_highlight` and `_score` to entities because answerfamily APIs don't do relay-style pagination, I had no choice but adding the field to entity itself.

Please consult `graphql/util/createConnectionType` for definition of edge objects:
- modify `createConnectionType` to add `highlight` to XXXEdges
- modify `defaultResolveEdges` to include _highlight